src/messages.ts

```diff
@@ -143,7 +143,10 @@ export type ExpectNever<T> = {[expectNever]: T; result: IsNever<T>}
  * @internal
  */
 const expectNullable = Symbol('expectNullable')
-export type ExpectNullable<T> = {[expectNullable]: T; result: Not<StrictEqualUsingBranding<T, NonNullable<T>>>}
+export type ExpectNullable<T> = {
+  [expectNullable]: T
+  result: Not<StrictEqualUsingBranding<T, NonNullable<T>, DeepBrandOptionsDefaults>>
```

Comment:
this is temp - we should make sure we respect whatever options the `.branded` that's calling this has mebe?? or should this not be using branding at all?

probably branding should always be opt-in since it can be slow